sxt/base/container/span_utility.h

```diff
@@ -45,4 +46,21 @@ span<T> subspan(Cont&& cont, size_t offset, size_t size) noexcept
 {
   return span<T>{cont}.subspan(offset, size);
 }
+
+//--------------------------------------------------------------------------------------------------
+// winked_span
+//--------------------------------------------------------------------------------------------------
+/**
+ * Convenience function for winked-out allocations.
+ *
+ * Note: Be sure to use this with a compatible allocator.
+ * See section 3 of https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0089r1.pdf
```

Comment:
Just to for my understanding of section 3, does a compatible monotonic allocator have to be derived from the `monotonic_buffer_resource` class? Similarly, does a compatible multipool allocator be derived from either the `synchronized_pool_resource` or  `unsynchronized_pool_resource` class? I think from [PR 61](https://github.com/spaceandtimelabs/blitzar/pull/61), a compatible allocation just has to be derived from `std::pmr::memory_resource`? Maybe I am misinterpreting what is meant by montonic and multipool allocators in section 3?

Reply:
It needs to derive from std::pmr::memory_resource and also deallocate everything upon descruction; otherwise, something like winked_span would leak memory